### PR TITLE
19428 Remove label truncation; Fix missing title attributes on labels

### PR DESCRIPTION
**Pull request by myabc**
https://community.openproject.org/work_packages/19428

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://myabc-fix-19428-label-truncation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-fix-19428-label-truncation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by myabc** [comment]
https://community.openproject.org/work_packages/19428

**Comment by NobodysNightmare** [comment]
Getting scrollbars when reducing the width further:

scrollbars

**Comment by NobodysNightmare** [comment]
Except my sighting from the WP form above, this PR looks nice :)

**Comment by myabc** [comment]
Marking WIP as I'd still like to fix this:
seeded_project_-_new_work_package_-_openproject

File: app/assets/stylesheets/content/_forms.sass [hunk 1]
```diff
@@ -244,6 +241,8 @@ fieldset.form--fieldset
 .form--label
   @include grid-content(2)
   @extend %label-style
+  overflow:     visible
+  overflow-y:   visible
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/content/_forms.sass:245]: Not sure how precendence stuff works here, but this looks weird.

Isn't `overflow` the same as `overflow-x` **and** `overflow-y`?

I'd either expect a one-liner: `overflow: visible` or when as a two-liner I'd expect:

``` CSS
overflow-x: visible;
overflow-y: visible;
```

**Comment by NobodysNightmare** [comment]
> Marking WIP as I'd still like to fix this:
> seeded_project_-_new_work_package_-_openproject

Have you thought about adding a `&nbsp;`? :grin: :grin: :grin:

**Comment by cratz** [comment]
> Have you thought about adding a `&nbsp;`?

Bad idea should be done through CSS

``` CSS
white-space: nowrap;
```

**Comment by myabc** [comment]
@cratz will be done w/CSS.

**Comment by cratz** [comment]
Hmm after I checked the screenshot (I know to late) it will not work with CSS.
Labels like "Start date" will stay on one line, except there is additional markup...

**Comment by NobodysNightmare** [comment]
@cratz WTF? Was the irony unclear? How many smiley faces do I have to add ^^

@myabc Have you an setmiate for the time required?

One could argue, that it is better to merge readable text (and readable star indicators) now and fix the minor wrapping issues later.

**Comment by cratz** [comment]
@NobodysNightmare I need at least 4 to recognize irony sorry.

**Comment by myabc** [comment]
@NobodysNightmare 

> One could argue, that it is better to merge readable text (and readable star indicators) now and fix the minor wrapping issues later.

Yep.

**Comment by NobodysNightmare** [comment]
Will have a final look at this PR then, and probably merge soon.

**Comment by NobodysNightmare** [comment]
Will merge this PR, as it is a big improvement over the current state. Still there is the (IMO) minor issue with wrapping "required" markers left.

I find it also disturbing that some field labels (e.g. responsible) already get overlapped by their input fields at a window width of ~1500px. When I throw the browser onto one half of my screen, I get < 1280 px and it looks like this:

text-cut

**Pull request merged**